scripts/create-protection-market-convex.ts

```diff
@@ -14,13 +14,29 @@ import chalk from 'chalk';
 import { getChainId, getContractAddress, getGasPrice, logSuccess, logFailure, findLog, waitForInput } from '../utils/utils'; // prettier-ignore
 import comptrollerAbi from '../abi/Comptroller.json';
 
+interface Pools {
+  [key: string]: PoolInfo;
+}
+
+interface PoolInfo {
+  contractName: string;
+  name: string;
+  symbol: string;
+  description: string;
+  underlyingName: string;
+  // Convex triggers with an underlying Curve LP token that use a Factory meta pool contract require the Curve base pool
+  // to be explicitly defined as it cannot be determined programmatically
+  basePool?: string;
+}
+
 // Constants
 const { AddressZero } = hre.ethers.constants;
 const cozyMultisig = '0x1725d89c5cf12F1E9423Dc21FdadC81C491a868b';
 
 // STEP 0: ENVIRONMENT SETUP
 const provider = hre.ethers.provider;
 const signer = new hre.ethers.Wallet(process.env.PRIVATE_KEY as string, hre.ethers.provider);
+// const [signer] = await hre.ethers.getSigners(); // for deploying to hardhat
```

Comment:
Usually for testing locally I temporarily import and call the `fundAccount` helper method in the utils file, that way we can test the deploy from the real deployer private key. This just helps minimize the differences between our test deploy and the real deploy

You can do this with `await fundAccount('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE', '5', signer.address, hre)` to fund the signer with 5 ETH

Also I don't think this line would work anyway since it's not in an async function?

Reply:
ahhh, I see. I was a bit stuck with funds with my real private key at first, but I see that `fundAccount` helper would resolve that. 

& yeah lol I had actually executed this within the `main` method. Removed.